packages/preview/wavy/0.1.0/typst.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "wavy"
+version = "0.1.0"
+entrypoint = "lib.typ"
+authors = ["Wenzhuo Liu"]
+license = "MIT"
+description = "Draw digital timing diagram in Typst using Wavedrom."
+
```

Comment:
could you exclude `wavg.svg`? The README itself should be included, but the example SVG isn't necessary (it's also not that small actually at 112KB).